src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -626,6 +637,9 @@ void RadioSpinel<InterfaceType, ProcessContextType>::HandleResponse(const uint8_
     VerifyOrExit(rval > 0 && cmd >= SPINEL_CMD_PROP_VALUE_IS && cmd <= SPINEL_CMD_PROP_VALUE_REMOVED,
                  error = OT_ERROR_PARSE);
 
+    otLogDebgPlat("Spinel resp: cmd:%s, prop:%s, iid:%02x, tid:%02x\n", spinel_command_to_cstr(cmd),
```

Comment:
The function `LogSpinelFrame()` has already parsed and logged the Spinel frame. This log may cause the duplicated logs. Can we update the `LogSpinelFrame()` to make it to log the `iid`?

Reply:
Good call. Updated the changes.